nodecontrol/algodControl.go

```diff
@@ -40,6 +40,22 @@ const StdErrFilename = "algod-err.log"
 // StdOutFilename is the name of the file in <datadir> where stdout will be captured if not redirected to host
 const StdOutFilename = "algod-out.log"
 
+type NodeAlreadyStoppedError struct {
+	algodDataDir string
+}
+
+func (e *NodeAlreadyStoppedError) Error() string {
+	return fmt.Sprintf("Node in directory %s has already stopped", e.algodDataDir)
+}
+
+type InvalidDataDirError struct {
```

Comment:
Travis failure: Comment is missing here.

`nodecontrol/algodControl.go:51:6: exported type InvalidDataDirError should have comment or be unexported`